CHANGES.md

```diff
@@ -1,3 +1,22 @@
+## edge-19.1.3
+
+* Controller
+  * Added `linkerd install-sp` command to install service profiles into the
+    linkerd control plane
+  * Improved API for ListPods (thanks @alenkacz!)
+  * fast-build has been improved and renamed to build-cli-bin
```

Comment:
wrap `fast-build` and `build-cli-bin`... also these maybe should go under a `Dev` section... or just remove? we don't usually include build tooling updates in release notes.